server/gendb/interface.go.tmpl

```diff
@@ -35,6 +35,8 @@ type Interface interface {
     {{- $indexName := indexName $xdbConfig.DynamoDB.KeySchema }}
     // Get{{ $modelName }}sBy{{ $indexName }} retrieves a list of {{ $modelName }}s from the database.
     Get{{ $modelName }}sBy{{ $indexName }}(ctx context.Context, input Get{{ $modelName }}sBy{{ $indexName }}Input) ([]models.{{ $modelName }}, error)
+    // Get{{ $modelName }}sBy{{ $indexName }}Page retrieves a page of {{ $modelName }}s from the database.
+    Get{{ $modelName }}sBy{{ $indexName }}Page(ctx context.Context, input Get{{ $modelName }}sBy{{ $indexName }}PageInput, fn func(m *models.{{ $modelName }}, last{{ $modelName }} bool) bool) error
```

Comment:
asked in Slack too, but I'm unsure about this API of passing a lambda not returning the slice